src/test/groovy/graphql/execution/ValuesResolverTest.groovy

```diff
@@ -100,7 +100,7 @@ class ValuesResolverTest extends Specification {
                 .type(GraphQLString)
         def inputType = newInputObject()
                 .name("Person")
-                .withAppliedDirective(Directives.OneOfDirective.toAppliedDirective())
+                .addAppliedDirective(Directives.OneOfDirective.toAppliedDirective())
```

Comment:
I updated tests to use the new "add" versions of builders, and I added a few legacy tests for coverage. We can delete the legacy tests when we removed the deprecated methods

Reply:
Scroll down to `AppliedDirectivesAreValid` test file to see the test that demonstrates the change in behaviour